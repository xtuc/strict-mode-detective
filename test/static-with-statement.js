function f(foo, values) {
  with (foo) {
    console.log(values);
  }
}
