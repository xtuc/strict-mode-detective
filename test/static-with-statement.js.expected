  1 | function f(foo, values) {
  2 |   with (foo) {
  3 |     console.log(values);
  4 |   }
  5 | }
  6 | 
'with' in strict mode (2:2)
